main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -243,14 +243,16 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
       DOM.loadHTML("core", "scripts/views/data-table/add-q-column-dialog.html"));
 
     var elmts = DOM.bind(frame);
-    elmts.dialogHeader.text($.i18n('core-views/add-id-col', column.name));
+    elmts.dialogHeader.text($.i18n('core-views/add-id-col', "\"" + column.name + "\""));
```

Comment:
Are we certain that such double quotes will be used in all languages ? Previously, you added those double quotes to the messages and it feels better to me. If you only update the English message for it, translators will be prompted to update the translation in their language, making it possible for them to review if double quotes are also appropriate or if they want to use other punctuation signs.

Reply:
The quotes definitely belong in the message. Any string concatenation in a user visible message is a "code smell," indicating something's probably wrong.

Reply:
I agree if we are going to use quotation marks, it should be in the message, since it would look different in some other languages like German and French.

I was thinking of using square brackets which seems more universal, for example:

Add new column containing entity identifiers on [Calcium]

So I switched to concatenating the quotation marks so it could be changed without having to update the translation.  Anyway, I can change it back.

There are several other places we are not distinguishing the column/entity name in a title string.